src/components/experiment-tracking/experiment-primary-toolbar/experiment-primary-toolbar.js

```diff
@@ -70,19 +62,13 @@ export const ExperimentPrimaryToolbar = ({
         visible={enableComparisonView}
         disabled={showChangesIconDisabled}
       />
-      <CSVLink
-        data={exportData}
-        asyncOnClick={true}
-        onClick={updateExportData}
-        filename="run-data.csv"
-      >
-        <IconButton
-          ariaLabel="Export graph as SVG or PNG"
-          className={'pipeline-menu-button--export'}
-          icon={ExportIcon}
-          labelText="Export run data"
-        />
-      </CSVLink>
+      <IconButton
+        ariaLabel="Export graph as SVG or PNG"
+        className={'pipeline-menu-button--export'}
```

Comment:
```suggestion
        className={'pipeline-menu-button--export-runs'}
```

Reply:
this is just from heap analytics perspective as Export button in Pipeline visualisation also has the exact same class name